@shared/constants/soroban/token.ts

```diff
@@ -26,3 +28,35 @@ export interface SorobanToken {
   symbol: string;
   decimals: number;
 }
+
+export type AssetsListKey = NETWORKS.PUBLIC | NETWORKS.TESTNET;
+
+export type AssetsLists = {
+  [K in AssetsListKey]: AssetsListItem[];
+};
+
+export interface AssetsListItem {
+  url: string;
+  enabled: boolean;
+}
+
+export const DEFAULT_ASSETS_LISTS: AssetsLists = {
+  [NETWORKS.PUBLIC]: [
+    {
```

Comment:
In the mocks, it looks like we will allow deleting a list but always want to retain one list as the "default" or one list that the user cannot delete. Should we add a field to the stellar expert list in order to key that one as the default one?

Reply:
Yeah, good question. I need to follow up with product on this. Not sure which of these should be "undeletable"